supervised/preprocessing/eda.py

```diff
@@ -162,3 +160,161 @@ def compute(X, y, eda_path):
             fout.close()
         except Exception as e:
             logger.error(f"There was an issue when running EDA. {str(e)}")
+
+    @staticmethod
+    def extensive_eda(X, y, save_path):
+
+        X = X.copy(deep=True)
+        # Check for empty dataframes in params
+        if not isinstance(X, pd.DataFrame):
+            raise ValueError("X should be a dataframe")
+        if X.shape[0] != len(y):
+            raise ValueError("X and y should have same number of samples")
+
+        if X.shape[1] > MAXCOL:
+            X = X.iloc[:, :MAXCOL]
+            logger.error(
+                f"AutoML EDA column limit exceeded! running for first {MAXCOL} columns"
+            )
+
+        ## replace * and / symbol for filenames
+        X.columns = [x.replace("*", "_").replace("/", "_") for x in X.columns]
```

Comment:
All forbidden characters should be replaced. Forbidden characters https://www.mtu.edu/umc/services/digital/writing/characters-avoid/

Reply:
Done.